packages/synapse-react-client/src/components/IconList.tsx

```diff
@@ -1,20 +1,42 @@
 import React from 'react'
 import IconSvg, { IconSvgProps } from './IconSvg/IconSvg'
 
+type IconConfigs = {
+  [index: string]: IconSvgProps // if the icon option has the "label" set, it will show tooltip in IconSvg
+}
+
 export type IconListProps = {
-  iconConfigs: {
-    [index: string]: IconSvgProps // if the icon option has the "label" set, it will show tooltip in IconSvg
-  }
+  iconConfigs: IconConfigs
   iconNames: string[]
+  iconFontSize?: string
```

Comment:
I was thinking we could just have `iconSx?: SxProps` instead of `iconFontSize`, but as I look at this, could you just pass this to `iconConfigs`?
```js
{
  sx: {
    fontSize: "40px"
  }
}